docs/taskruns.md

```diff
@@ -328,12 +342,11 @@ Status:
 
 ```
 
-Results will be printed verbatim; any new lines or other whitespace returned as part of the result will be included in the output.
-
-## Cancelling a TaskRun
+## Cancelling a `TaskRun`
 
-In order to cancel a running task (`TaskRun`), you need to update its spec to
-mark it as cancelled. Running Pods will be deleted.
+To cancel a `TaskRun` that's currently executing, update its definition
```

Comment:
nit - s/that's/that is

Reply:
I think Tekton docs and Google's own style guide allows for contractions, it sounds less stiff.